src/goto-programs/goto_functions_template.h

```diff
@@ -102,16 +102,23 @@ class goto_functions_templatet
   typedef goto_function_templatet<bodyT> goto_functiont;
   typedef std::map<irep_idt, goto_functiont> function_mapt;
   function_mapt function_map;
+  /// A location number such that unused_location_number - MAX_UINT are all
```

Comment:
Is that `-` a minus (subtraction) or (more likely) indicating a range? Would you mind being clinical here and use a notation that unambiguously states whether that's a (half) open interval?